streams/src/main/java/org/apache/kafka/streams/StoreQueryParams.java

```diff
@@ -21,12 +21,11 @@
 import java.util.Objects;
 
 /**
- * Represents all the query options that a user can provide to state what kind of stores it is expecting.
- * The options would be whether a user would want to enable/disable stale stores
- * or whether it knows the list of partitions that it specifically wants to fetch.
- * If this information is not provided the default behavior is to fetch the stores for all the partitions
+ * Represents all the query options that a user can provide to control the kind of stores we would like to fetch.
+ * The options could indicate - whether we want stale stores to be included (or)
+ * we want to filter for only stores belonging to a partition.
+ * If no specific partition is specified the default behavior is to fetch the stores for all the partitions
  * available on that instance for that particular store name.
```

Comment:
This comment can easily become outdated as the class evolved. Better to keep it high level and defer to the setters' javadocs for a description of the available operations.

Eg:
`{@code StoreQueryParams} allows you to pass a variety of parameters when fetching a store for interactive query.`

Reply:
Done.